modules/builds-using-build-volumes.adoc

```diff
@@ -77,12 +88,23 @@ spec:
           type: ConfigMap <3>
           configMap:
             name: my-config <4>
+      - name: my-csi-volume <1>
+        mounts:
+        - destinationPath: /opt/app-root/src/some_path  <2>
+        source:
+          type: CSI <3>
+          csi:
+            driver: csi.driver.ephemeral.io <5>
+            volumeAttributes: <6>
+              attribute: value
 ----
 
 <1> Required. A unique name.
 <2> Required. The absolute path of the mount point. It must not contain `..` or `:` and doesn't collide with the destination path generated by the builder. The `/opt/app-root/src` is the default home directory for many Red Hat S2I-enabled images.
-<3> Required. The type of source, `ConfigMap` or `Secret`.
+<3> Required. The type of source, `ConfigMap`, `Secret` or `CSI`.
 <4> Required. The name of the source.
+<5> Required. The driver that provides the ephemeral CSI volume.
+<6> Optional. The volume attributes of the ephemeral CSI volume.
```

Comment:
@gabemontero @jkhelil Do these two callouts provide this enough information for a newer user to successfully enter those values, or do we need to provide additional guidance, such as links to other topics?

Reply:
> did you mean @jkhelil instead of @akram (this is @jkhelil 's PR)?

Sorry about that. I meant @jkhelil. Thanks.

Reply:
hey @rolfedh Yeah i think it would be nice to have a link to how to add  ephemeral csi volume to a pod spec, i cant find that doc, wondering if it exist

Reply:
@jkhelil It looks like [Embedding a CSI inline ephemeral volume in the Pod specification](https://docs.openshift.com/container-platform/4.6/storage/container_storage_interface/ephemeral-storage-csi-inline.html#ephemeral-storage-csi-inline-pod_ephemeral-storage-csi-inline) is what we're looking for. If so, you can add a cross-reference like this to the module: 
```
xref:../../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc#ephemeral-storage-csi-inline-pod_ephemeral-storage-csi-inline[Embedding a CSI inline ephemeral volume in the Pod specification]
```

Reply:
done @rolfedh please have a look

Reply:
@rolfedh asked to remove xref at the end